lib/insert/insert_donation.rb

```diff
@@ -173,7 +173,7 @@ def self.insert_payment(kind, fee_total, donation_id, data)
   def self.insert_donation(data, entities)
     d = Donation.new
     d.date = data['date']
-    d.anonymous = data['anonymous']
+    d.anonymous = data['anonymous'].nil? ? false : data['anonymous']
```

Comment:
I think it'd be more logical if it was in a `:before_save`.

You might consider doing : `data['anonymous'] || false` But I'm not sure that's actually as clear.